guides/micronaut-aws-secretsmanager-rotation/java/src/main/java/example/micronaut/Handler.java

```diff
@@ -0,0 +1,95 @@
+package example.micronaut;
+
+import com.amazonaws.services.lambda.runtime.events.SecretsManagerRotationEvent;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.micronaut.aws.distributedconfiguration.KeyValueFetcher;
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.function.aws.MicronautRequestHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.secretsmanager.SecretsManagerClient;
+import software.amazon.awssdk.services.secretsmanager.model.PutSecretValueRequest;
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+@Introspected
+public class Handler
+        extends MicronautRequestHandler<SecretsManagerRotationEvent, Void> { // <1>
+
+    private static final Logger LOG = LoggerFactory.getLogger(Handler.class);
+
+    private static final String JWK_PRIMARY = "jwk.primary";
+    private static final String JWK_SECONDARY = "jwk.secondary";
+
+    @Inject
+    public JsonWebKeyGenerator jsonWebKeyGenerator; // <2>
+
+    @Inject
+    public ObjectMapper objectMapper; // <3>
+
+    @Inject
+    public SecretsManagerClient secretsManagerClient; // <4>
+
+    @Inject
+    public KeyValueFetcher keyValueFetcher; // <5>
+
+    @Override
+    public Void execute(SecretsManagerRotationEvent input) {
+        if (LOG.isInfoEnabled()) {
```

Comment:
no need for is-enabled check unless either `input.getStep()` or `input.getSecretId()` is expensive

Reply:
I had been using the same convention as in the modules to always check. I will remove the if statements.